geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/data/RedisSortedSetTest.java

```diff
@@ -278,6 +279,84 @@ public void zrange_shouldAlsoReturnScores_whenWithScoresSpecified() {
         "member6".getBytes(), "1.5".getBytes());
   }
 
+  @Test
+  public void zlexcount_shouldBeInclusiveWhenSpecified() {
+    RedisSortedSet sortedSet = createRedisSortedSet(
+        "1.1", "member1",
+        "1.1", "member2",
+        "1.1", "member3",
+        "1.1", "member4",
+        "1.1", "member5");
+    SortedSetLexRangeOptions lexOptions =
+        new SortedSetLexRangeOptions("[member1".getBytes(), "[member3".getBytes());
+    assertThat(sortedSet.zlexcount(lexOptions)).isEqualTo(3);
+  }
+
+  @Test
+  public void zlexcount_shouldBeExclusiveWhenSpecified() {
+    RedisSortedSet sortedSet = createRedisSortedSet(
+        "1.1", "member1",
+        "1.1", "member2",
+        "1.1", "member3",
+        "1.1", "member4",
+        "1.1", "member5");
```

Comment:
looks like you're passing in the same values in all these tests that you added. might be worth abstracting that out. I don't think that having that there is crucial for test readability

Reply:
Done